INTERFACE_SPECIFICATION.md

```diff
@@ -0,0 +1,121 @@
+## The Bitcoin Canister API
+
+The canister ID of the Bitcoin canister for Bitcoin mainnet is `ghsi2-tqaaa-aaaan-aaaca-cai`.\
+The canister ID of the Bitcoin canister for Bitcoin testnet (v4) is `g4xu7-jiaaa-aaaan-aaaaq-cai`.
```

Comment:
NIT: and additionally maybe to add links to corresponding dashboards where you can clearly see all the public API methods with interface and may also call methods from that page

https://dashboard.internetcomputer.org/canister/ghsi2-tqaaa-aaaan-aaaca-cai

Reply:
Good idea!

Reply:
Done!